### ISSUE TITLE
Reduce log level to WARN or INFO: Error loading custom configuration file '/etc/hirte/agent.conf': 'No such file or directory'.

### ISSUE THREAD

**Issue by dougsland** [engelmi/hirte#299]
## Describe the bug

Hirte daemon and agent in recent builds now looks for /etc/hirte/hirte.conf.d/hirte.conf and /etc/hirte/agent.conf.d/agent.conf as another place for configs. However, it still looks for /etc/hirte/hirte.conf and /etc/hirte/agent.conf. If unable to find for example `/etc/hirte/hirte.conf` or `/etc/hirte-agent/agent.conf` it shows an error message.  Most of the cases when a software show an error is something bad, in this case it's not that bad, just a "info" like message as the software continue operate normally.

Please consider changing to change the message to **WARN** or **INFO**.  
Another approach is send a message to users like (in positive way): "Found a config /etc/hirte/hirte.conf.d/hirte.conf loading..."
In a negative way: Error: Unable to find a valid hirte config file in any of the following paths: /etc/hirte/hirte.conf, /etc/hirte/hirte.conf.d/hirte.conf, /usr/share/hirte.conf/hirte.conf. Exiting..."

## More info
Examples:
```
May 11 03:07:47 control systemd[1]: Started Hirte systemd service controller manager daemon.
May 11 03:07:47 control hirte[767]: Error loading custom configuration file '/etc/hirte/hirte.conf': 'No such file or directory'.
``` 

```
May 10 21:26:29 node1 hirte-agent[519]: Error loading custom configuration file '/etc/hirte/agent.conf': 'No such file or directory'.
```

Last test executed in **hirte** and **hirte-agent** in version 0.3.0-0.202305101100.git738d22e.el9.x86_64

See also:
```
commit 9b480ee5e88e7e0648ddd02a43d3f8cc154fbc6f
Author: Daniel J Walsh <dwalsh@redhat.com>
Date:   Wed May 10 04:01:45 2023 -0400

    hirte config files should be called the same everywhere (#253)

    hirte-agent should read the agent.conf file in /usr and optionally
    in /etc.

    hirte should read the hirte.conf file in /usr and optionally in /etc.

    Only install hirte.conf and agent.conf once.  Distros should install in
    /usr/share/hirte/config/ and admins can copy it to /etc/hirte/

    Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>
```

**Comment by engelmi** [comment]
If I understand it correctly, then the change to use `%ghost` does not install the `/etc/hirte/agent.conf` and `/etc/hirte/hirte.conf` anymore - so it is normal that these files could be missing. 
I created PR (#300) to address this issue. Could you have a look at it? @dougsland

**Comment by engelmi** [comment]
Fixed by #300. Closing.
